src/eredis_client.erl

```diff
@@ -223,10 +224,14 @@ handle_info(_Info, State) ->
 terminate(_Reason, State) ->
     close(State#state.socket).
 
-%% Down
 code_change(_, State, _) ->
     {ok, State}.
 
+format_status(_Opt, [_PDict, #state{} = State]) ->
+    [{data, [{"State", State#state{password = "******"}}]}];
+format_status(_Opt, [_PDict, State]) ->
+    [{data, [{"State", State}]}].
```

Comment:
Nit: it seems that `format_status/2` was deprecated in favor of `format_status/1`: https://www.erlang.org/doc/man/gen_server.html#Module:format_status-2

Reply:
Never mind, it's only in OTP 25 which we're not using yet.